### PR TITLE
Added meshery operator project logo and description

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: anita-ihuman <charlesanita403@gmail.com>

**Description**

This PR fixes #536 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,13 @@ alt="Image Hub" align="left" /></a>
 <a href="https://layer5.io/projects/image-hub">Image Hub</a> is a demo application which runs on <a href="https://meshery.layer5.io/docs/service-meshes/adapters/consul">Consul</a> and facilitates exploring WebAssembly modules used as Envoy Filters.
 <br /><br /><br />
 </p>
-
+</p>
+<p style="clear:both;">
+<h2><a href="https://github.com/layer5io/meshery-operator">Meshery Operator</a></h2>
+<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="img/readme/meshery-operator-dark.svg" style="margin:10px;" width="150px" align="left" /></a>
```
Comment (kushthedude) [inline README.md:62]: @Anita-ihuman Can you please fix the logo src here?

Comment (Anita-ihuman) [reply README.md:62]: whats wrong with it, please? please point the error

Comment (kushthedude) [reply README.md:62]: There is no `img/readme/meshery-operator-dark.svg` file

**Comment by kushthedude** [comment]
Way to go @Anita-ihuman 🥳 🥳 !!

**Comment by leecalcote** [comment]
Whoo-hoo! @Anita-ihuman 😄

**Pull request merged**